--- bro-2.5.1/CMakeLists.txt	2017-10-08 09:40:25.264288268 -0500
+++ bro-2.5.1/CMakeLists.txt.new	2017-10-08 09:41:17.247937581 -0500
@@ -25,7 +25,7 @@
 get_filename_component(BRO_SCRIPT_INSTALL_PATH ${BRO_SCRIPT_INSTALL_PATH}
     ABSOLUTE)

-set(BRO_PLUGIN_INSTALL_PATH ${BRO_ROOT_DIR}/lib/bro/plugins CACHE STRING "Installation path for plugins" FORCE)
+set(BRO_PLUGIN_INSTALL_PATH ${BRO_ROOT_DIR}/lib/bro/plugins CACHE STRING "Installation path for plugins")

 configure_file(bro-path-dev.in ${CMAKE_CURRENT_BINARY_DIR}/bro-path-dev)

@@ -173,7 +173,7 @@
 set(DEFAULT_BROPATH .:${BRO_SCRIPT_INSTALL_PATH}:${BRO_SCRIPT_INSTALL_PATH}/policy:${BRO_SCRIPT_INSTALL_PATH}/site)

 if ( NOT BINARY_PACKAGING_MODE )
-    set(BRO_DIST ${CMAKE_SOURCE_DIR})
+    set(BRO_DIST ${CMAKE_SOURCE_DIR} CACHE STRING "Path for Bro development distribution files." )
 endif ()

 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/bro-config.h.in
